.travis.yml

```
@@ -20,6 +20,13 @@ jobs:
         - PHPUNIT_TEST=1
         - PHPUNIT_SUITE="framework"
         - COMPOSER_INSTALL_ARG="--prefer-lowest"
+    - php: 7.4
+      env:
+        - DB=PGSQL
+        - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
+        - PHPCS_TEST=1
```

```suggestion
        - PHPCS_TEST=1
```
We're already doing PHPCS_TEST above

---

Fixed.